gcc/rust/expand/rust-macro-builtins-asm.cc

```diff
@@ -0,0 +1,21 @@
+//
+// Created by jjsm on 2/25/24.
+//
+
+#include "rust-macro-builtins-asm.h"
+
+namespace Rust {
+const BiMap<std::string, BuiltinMacro> asm_macros = {{
+  {"asm", BuiltinMacro::Asm},
+  {"llvm_asm", BuiltinMacro::LlvmAsm},
+  {"global_asm", BuiltinMacro::GlobalAsm},
+}};
+
+const std::unordered_map<std::string, AST::MacroTranscriberFunc>
+  asm_builtin_transcribers = {
+    {"asm", MacroBuiltin::sorry},
+    {"llvm_asm", MacroBuiltin::sorry},
+    {"global_asm", MacroBuiltin::sorry},
+};
```

Comment:
I like this idea of splitting the maps, but I think it might also make code harder to read and follow. I think it would be better to keep the maps all in one place if possible - either in `rust-macro-builtins.h` or `rust-macro-builtins.cc`

Reply:
so keep both maps as they originally were, but split off the handlers into files as you've done

Reply:
gotcha